app/src/main/java/helper/ModulesAdapter.java

```diff
@@ -185,7 +189,21 @@ public void onClick(DialogInterface dialogInterface, int i) {
                                     mFileManager.downloadFile(module.getContents().get(0), module, courseName);
                                     break;
                                 case 1:
+                                    String toShare = "";
```

Comment:
Don't put the business logic here. Please create a private function and call it inside the switch-case.